index.html

```diff
@@ -8,12 +8,96 @@
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@100&display=swap" rel="stylesheet">
 </head>
 <body>
+    <header class="header">
+        <!-- logo and nav goes here -->
+        <!-- ask for later about line 19.. why it went back to huge logo -->
 
+        <a href="#">
+            <div class="header__logo">
+                
+              <img src="img/karma-logo.svg"  alt="logo of karma">
```

Comment:
Cosmetics: We won't need any new lines around the `<img>` tag because it's already inside the `div` group and we usually put the together (or group them together) without any extra new line.